template-compose/app/build.gradle.kts

```diff
@@ -1,33 +1,33 @@
 import org.jetbrains.kotlin.konan.properties.loadProperties
 
 plugins {
-    id(Plugins.ANDROID_APPLICATION)
-    id(Plugins.KOTLIN_ANDROID)
-    id(Plugins.KOTLIN_KAPT)
-    id(Plugins.KOTLIN_PARCELIZE)
-    id(Plugins.HILT_ANDROID)
-    id(Plugins.KOVER)
+    alias(libs.plugins.android.application)
+    alias(libs.plugins.kotlin.android)
+    alias(libs.plugins.kotlin.kapt)
```

Comment:
@luongvo We need to migrate kapt to ksp also (Creating new issue to handle this). Hdyt?

Reply:
@hoangnguyen92dn yes, please. It should be the following task after https://github.com/nimblehq/android-templates/issues/587. Please refer your done work in GT as reference 🙏

Reply:
I think we could add one more AC to the issue created by Ryan above. it's simple following the guideline: https://developer.android.com/build/migrate-to-ksp

Reply:
Agree +1